### DISCUSSION TITLE
Add OpenHaystack compatibility

### DISCUSSION THREAD

**Discussion by stepjanssen** [MartinPham/FindMySync#2] [Ideas]
[OpenHaystack](https://github.com/seemoo-lab/openhaystack) is a great tool to create ESP32 based Find My devices. An [ESPHome component](https://www.barrenechea.cl/projects/openhaystack-esphome/) has also been created for it. Unfortunately, OpenHaystack seems to have been abandoned and no longer works on the latest MacOS versions. It also didn't have long term tracking abilities like FindMySync.

Adding OpenHaystack to FindMySync would make it a great tool for low budget tracking of cars or pretty much anything else!

**Comment by MartinPham** [comment]
Thanks! Let me try
